### DISCUSSION TITLE
Snapshot builds

### DISCUSSION THREAD

**Discussion by juanpabloprado** [awslabs/aws-mysql-jdbc#193] [Q&A] [answered]
Are there any snapshot builds on a mvn repository so that we could try https://github.com/awslabs/aws-mysql-jdbc/commit/5b937fc9f6b9832a5037840001bb130606ee8761 before next release

**Comment by sergiyvamz** [comment, marked as answer]
Hello @juanpabloprado 

You can find our snapshot builds at 
https://aws.oss.sonatype.org/service/local/repositories/snapshots/content/software/aws/rds/aws-mysql-jdbc/maven-metadata.xml
